docs/specification/draft/server/profiles.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Profiles
+---
+
+<div id="enable-section-numbers" />
+
+<Info>**Protocol Revision**: draft</Info>
+
+The Model Context Protocol enables the definition of a collection of behaviors and properties as Profiles. Standard Profiles are available through the Model Context Protocol, and anyone may define custom Profiles. MCP Servers can declare the profiles they support. MCP Clients can discover which profiles an MCP Server supports, request the use of specific profiles, and negotiate a mutually agreed profile to use within a session with the MCP Server.
```

Comment:
As with the comment in the minimum authorization profile, it's unclear _why_ profiles are needed, who consumes them, and what decisions are made based on them. You kind of allude to it:

>MCP Clients can discover which profiles an MCP Server supports

But what is the scenario under which a client needs to discover a profile instead of attempt specific interactions (e.g., auth) and then using the metadata for decision-making?